Block Reward Increase with scheduled decrease to 1ETH BR - 3368.md (#3368)
ethereum/EIPs: EIPS/eip-3333.md

---

> eip: 3333

Reviewer: ```suggestion
eip: 3368
```

Reviewer: Note: filename needs to be updated as well to reflect this.

> title: Increase block rewards to 3 ETH, with 2 Year Decay to 1 ETH Scheduled

Reviewer: ```suggestion
title: Set block reward to 3 ETH with a 2 Year Decay to 1 ETH
```
Try to phrase things throughout the EIP in the context of a future reader who may not have a knowledge of historical state, and also so they can be implemented without needing to know previous state (e.g., this could be implemented on a chain that currently has a 5 ETH block reward or a chain with a 1 ETH block reward).

Author: Copy, good change

> author: Michael D. Carter (BitsBeTrippin)

Reviewer: ```suggestion
author: Michael D. Carter (@bitsbetrippin)
```

> discussions-to: https://discord.gg/3sjBbWKM4E (ETH R&D 1559-mining)

Reviewer: Discussions-to links need to be to a semi-permanent asynchronous discussion forum.  A thread at https://ethereum-magicians.org/ is strongly preferred, though a new issue in this GitHub repository is considered an acceptable alternative if you unable or unwilling to create a thread over there.

Author: Will add

status: Draft
type: Standards Track
category: Core
created: 2021-03-11
---

## Simple Summary
Changes the block reward paid to proof-of-work (POW) miners to 3 ETH from existing 2 ETH, but kicks off a 0.25 quarterly drop to a final 1.0 ETH Reward 2 Years Later.

## Abstract

> Starting with `FORK_BLKNUM` block rewards will be increased to a base of 3 ETH, uncle and nephew rewards will be adjusted accordingly. Every 3 months after, measured by EPOCH count, Block Reward drops 0.25 every quarter till Ethereum Block Reward reaches 1.0 BR per Block.

Reviewer: See comments below about making this smooth rather than having cliffs every 3 months.  If that change is accepted, this should be updated to reflect that.

Author: Accepted, makes more sense

## Motivation

> With the passing of EIP-1559, the potential steep reduction of Ethereum Miner fee's, which currently are averaging a doubling of the exiting block reward have several scenerios where it can become the 50% key factor on keeping ethereum mining participation growing the current all time high hashpower, all the way to the London hard fork release. There are several scenerios where this can create a steep hashrate decrease when the BASEFEE starts to get burned vs distributed to miners. This can create an exteciential risk to ethereum's network by incentivizing miners to move to more profitable solutions, including broker services such as Nicehash and Miningrigrentals. To ward off this threat, a recommend action is to temporarly increase Eth Mining Rewards by +1 offset then began to decrease -2 Eth over 8 quarters, 2 full years from London's Hardfork. This should align well with the current optimisitic ETH 2.0 PoW/PoS handoff and appropriately set Eth Inflation on a managed reduction against a schedule that can be planned for. Additionally this provides a higher incentive for price discovery which can offset any further steep impacts for the mining security budget. 

Reviewer: Try to avoid mentioning London or other things related to mainnet or current state of Ethereum in EIPs.  Consider that an EIP may be implemented at any time in the future.  Perhaps reword this to say something along the lines of:

> A sudden drop in PoW mining rewards could result in a sudden precipitous decrease in mining profitability that may drive miners to auction off their hashrate to the highest bidder while they figure out what to do with their now "worthless" hardware.  If enough hashrate is auctioned off in this way at the same time, an attacker will be able to rent a large amount of hashing power for a short period of time at relatively low cost and potentially attack Ethereum.  By setting the block reward to X (where X is enough to offset the sudden profitability decrease) and then decreasing it over time to Y (where Y is a number below the sudden profitability decrease), we both avoid introducing long term inflation while at the same time spreading out the rate that individual miners cross into the unprofitable range.  This allows miners time to sell off/repurpose their hardware rather than a large amount of hashing power being put up for auction at the same time while people try to figure out what to do with their now "worthless" hardware.

Author: Good add

> ## Specification
> #### Adjust Block, Uncle, and Nephew rewards
> Adjust the block reward to `new_block_reward`, where
> 
>     new_block_reward = 3_000_000_000_000_000_000 if block.number >= FORK_BLKNUM else block.reward
> 
> (5E18 wei, or 3,000,000,000,000,000,000 wei, or 3 ETH).
> 
> Analogue, if an uncle is included in a block for `block.number >= FORK_BLKNUM` such that `block.number - uncle.number = k`, the uncle reward is
> 
>     new_uncle_reward = (8 - k) * new_block_reward / 8
> 
> This is the existing formula for uncle rewards, simply adjusted with `new_block_reward`.
> 
> The nephew reward for `block.number >= FORK_BLKNUM` is
> 
>     new_nephew_reward = new_block_reward / 32
> 
> This is the existing formula for nephew rewards, simply adjusted with `new_block_reward`.
> 

Reviewer: ````suggestion
## Specification
### Constants
* `TRANSITION_START_BLOCK_NUMBER: TBD`
* `TRANSITION_DURATION: 8_000_000` (about two years)
* `TRANSITION_END_BLOCK_NUMBER: FORK_BLOCK_NUMBER + TRANSITION_DURATION`
* `STARTING_REWARD: 3_000_000_000_000_000_000`
* `ENDING_REWARD: 1_000_000_000_000_000_000`
* `REWARD_DELTA: STARTING_REWARD - ENDING_REWARD`

### Block Reward
```py
if block.number >= TRANSITION_END_BLOCK_NUMBER:
    block_reward = ENDING_REWARD
elif block.number >= TRANSITION_START_BLOCK_NUMBER:
    block_reward = STARTING_REWARD - REWARD_DELTA * TRANSITION_DURATION / (block.number - TRANSITION_START_BLOCK_NUMBER)
```
````
A smooth transition is generally preferable to cliffs, for the exact reasons outlined in the Motivation section.  😄 

Also, no need to mention uncle or nephew rewards since those formulas aren't changing.

## Rationale

> A 3 ETH base reward with quarterly schedule on decrease. 0.25 reduction per 3 months for 2 years to a total reduction of -2 Eth.

Reviewer: The Rational section is for discussing why specific decisions were made.  Consider writing a **very** brief statement about why 2 years was chosen as the transition duration here.
```suggestion
2 years was chosen because it gives miners sufficient time to find alternative uses for their hardware and/or get their hardware back out onto the open market (e.g., in the form of gaming GPUs) without flooding the market suddenly.
```

Author: accepted
